doc/whats_new/v0.20.rst

```diff
@@ -179,6 +179,14 @@ Metrics
   :issue:`10093` by :user:`alexryndin <alexryndin>`
   and :user:`Hanmin Qin <qinhanmin2014>`.
 
+  Neighbors
```

Comment:
The [doc](https://14971-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/whats_new/v0.20.html) does not look fine, please follow what others are doing (not sure but maybe remove some extra whitespace at the beginning?).